internal/util/telemetry/telemetry.go

```diff
@@ -24,6 +24,13 @@ import (
 	"go.uber.org/zap"
 )
 
+// Telemetry states.
+const (
+	Enabled   string = "enabled"
+	Disabled  string = "disabled"
+	Undecided string = "undecided"
+)
```

Comment:
We already have a representation for those: true, false, and nil. No need to add another one

Reply:
@rumyantseva WDYT?

Reply:
@AlekSi I don't understand your comment. We don't use true, false, or nil in the places where we use these particular constants.

Reply:
Those constants are used twice in the code, so maybe it's ok to leave them? Also, we can use those constants to compose the message like  "monitoring is " + state.

Reply:
(overall, I'm ok with whatever can help us finish this task asap)

Reply:
> We don't use true, false, or nil in the places where we use these particular constants.

Those constants represent things that already have a different representation. We don't use them in the state, for example.

> Those constants are used twice in the code, so maybe it's ok to leave them?

By that logic we should have a constant for "ok" string that is used everywhere.

Reply:
I reverted the change. All state's string representations are called without any const now

Reply:
@AlekSi as someone who uses things like `http.MethodGet` instead of `"GET"` I don't feel it's bad to have a constant for `ok` :D But I got the point, yeah.

@noisersup thanks for reverting the change!